src/core/unit.h

```diff
@@ -114,6 +114,7 @@ struct Unit {
 
         /* Job timeout and action to take */
         usec_t job_timeout;
+        usec_t job_running_timeout;
```

Comment:
The two (finite) timeouts will always have a common end. JobTimeoutSec= is
spanning the JobRunningTimeoutSec= so there is only one timer event created for
the timeout that goes off earlier. (I find two events couter-productive if
equal/close values were configured for both.)